CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Cromwell Change Log
 
+## 34 Release Notes
+
+### Requester Pays on GCS
+
+Access of Google Cloud Storage that have Requester Pays enabled is now supported.
+Please read the [relevant documentation](http://cromwell.readthedocs.io/en/develop/filesystems/GoogleCloudStorage#requester-pays) for information on how to enable it and the consequences.
```

Comment:
Hmm no idea, what does stable correspond to ?

Reply:
I believe @katevoss changed that so the docs we publish correspond to current releases and not unreleased code on develop.

Reply:
Ok so this doc won't be on `stable` until we release ? If I change the link to point to `stable` it will be broken then no ?

Reply:
From [Read The Docs](https://docs.readthedocs.io/en/latest/versions.html#how-we-envision-versions-working): 
> *How we envision versions working*
> In the normal case, the latest version will always point to the most up to date development code. If you develop on a branch that is different than the default for your VCS, you should set the Default Branch to that branch.

Since Cromwell develops on the develop branch, there is a [`/develop` version of this docs](http://cromwell.readthedocs.io/en/develop/). This version of the docs corresponds to the develop branch of the code. When someone merges their code to develop, the `/develop` docs are updated.

> We (Write the Docs) also create a `stable` version, if your project has any tagged releases. `stable` will be automatically kept up to date to point at your highest version.
> You should push a tag for each version of your project. These tags should be numbered in a way that is consistent with semantic versioning. This will map to your stable branch by default.

@Horneth is correct that this doc won't be on `/stable` until you release that version. However the doc will be on `/develop` once it is on the develop branch in Github. 

I hope this clarifies the doc situation, let me know if I can be of any more help/hindrance.